### DISCUSSION TITLE
How do I skip downloading all files with a certain extension? (What arguments does --skip accept?)

### DISCUSSION THREAD

**Discussion by ymgenesis** [Serene-Arc/bulk-downloader-for-reddit#331] [Q&A] [answered]
Trying to get bdfr to skip downloading all .txt files. Tried using --skip txt, --skip .txt, --skip *txt, to no avail.

How can I get this to work? Thanks for the help.

**Comment by Serene-Arc** [comment, marked as answer]
It should work with you specifying `txt` or `.txt`. If it isn't working with that, could you open an issue with a submission that should be skipped but isn't so I can investigate?

**Reply by ymgenesis** [reply]
Sure, thanks.
